ambry-commons/src/test/java/com.github.ambry.commons/HelixPropertyStoreUtils.java

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.commons;
+
+import com.github.ambry.config.HelixPropertyStoreConfig;
+import com.github.ambry.config.VerifiableProperties;
+import java.util.ArrayList;
+import java.util.Properties;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.atomic.AtomicReference;
+import org.apache.helix.AccessOption;
+import org.apache.helix.ZNRecord;
+import org.apache.helix.store.HelixPropertyListener;
+import org.apache.helix.store.HelixPropertyStore;
+
+import static com.github.ambry.commons.TestUtils.*;
+
+
+/**
+ * Utilities for operating on the corresponding part of a {@code ZooKeeper} for a {@link HelixPropertyStore}.
+ */
+class HelixPropertyStoreUtils {
+  private static final long operationTimeoutMs = 20000;
```

Comment:
This setting was copied from helix test code. Then I looked into the default value for instantiating a `HelixManager`, which seems to use a longer timeout value.
`DEFAULT_CONNECTION_TIMEOUT = 60 * 1000 and DEFAULT_SESSION_TIMEOUT = 30 * 1000`. I guess 20s or 30s is in a reasonable range.